src/server/scripts/Outland/BlackTemple/boss_teron_gorefiend.cpp

```diff
@@ -91,7 +91,7 @@ class npc_doom_blossom : public CreatureScript
 
         void Despawn()
         {
-            me->DealDamage(me, me->GetHealth(), NULL, DIRECT_DAMAGE, SPELL_SCHOOL_MASK_NORMAL, NULL, false);
+            me->DealDamage(me, me->GetHealth(), nullptr, DIRECT_DAMAGE, SPELL_SCHOOL_MASK_NORMAL, nullptr, false);
```

Comment:
is not my main intention fix/remove this hacks
almost scripts in black temple, need rewrite.
Someday in future, when i have free time and no one already did this, i will rewrite boss per boss
But for now, my intention is add support for boundary in shade of akama :smile: